dosagelib/plugins/b.py

```diff
@@ -167,6 +167,38 @@ class BillyTheDunce(_ParserScraper):
     endOfLife = True
 
 
+class BirdBoy(_WordPressScraper):
+    url = 'https://bird-boy.com/'
+    stripUrl = url + 'comic/{0}-{1}/'
+    firstStripUrl = stripUrl.format('volume-i', 'the-sword-of-mali-mani')
+    help = 'volume-page # Examples: i-the-sword-of-mali-mani, iii-7, synopsis-2'
+
+    def getIndexStripUrl(self, index):
+        (volume, strip) = index.split('-', maxsplit=1)
+        try:
+            pageNr = int(strip)
+        except ValueError:
+            pageNr = None # Use the string to fetch a cover page
+        if volume == 'synopsis':
+            strip = '{0}{1}'.format(pageNr, '-02' if strip in [1,3] else '')
+        else:
+            volume = 'volume-' + volume
+            if pageNr is not None:
+                strip = 'page-{0}'.format(pageNr)
+        return self.stripUrl.format(volume, strip)
```

Comment:
Not sure if using such a complex function is worth the hassle... Most modules just let the user specify part of the URL and be done with it...

Reply:
I'm fine with whatever. Most annoying part is the two synopsis pages that break with the pattern. Just let me know and I'll drop it.

Reply:
I'll just keep it as-is for now, not that important :D